src/set/set.service.ts

```diff
@@ -36,6 +38,7 @@ export class SetService {
             const set = (
                 await this.setModel.create({
                     ...createSetDto,
+                    slug: slugged(createSetDto.name),
```

Comment:
is this the normal name used for this? Wouldn´t it make more sense to name the methode "slugify"?

Reply:
Renamed to createSlug